wix-bookings/guides/Terminology.md

```diff
@@ -1,58 +1,57 @@
+| Policy (Bookings) | Rules set by the merchant regarding booking and canceling a service. These may include how far in advance customers can book, the deadline for cancellations, and the maximum number of participants allowed per booking. |
+| Pricing provider | A 3rd-party app that implements custom logic to determine the price of a booking with the [Pricing Integration Service Plugin](https://dev.wix.com/docs/rest/business-solutions/bookings/pricing/pricing-integration-spi/introduction). |
+| Resource | A business asset. For example, a staff member, a room, or equipment that's needed to provide a service. Each resource has a schedule that defines its availability. |
+| Schedule | A collection of all sessions associated with a specific service or resource. It includes information necessary to calculate availability. For more details, see appointment schedule, class schedule, and course schedule. |
+| Service | A business offering, that can be a class, course, or appointment-based. For example, a fitness studio may offer 30-minute 1-on-1 personal training sessions by appointment, 1-hour yoga classes every Monday at 7:00 PM, and introductory HIIT courses consisting of 3 training sessions on specific dates. For more information, see appointment, class, and course. |
+| Status (Booking) | Information about the life cycle status of the booking. Statuses include: `CREATED`, `PENDING`, `CONFIRMED`, `DECLINED`, `WAITING_LIST`, and `CANCELED`. |
+| Session | A booked or reserved period of time for an appointment, class, or course. Alternatively, you can consider a session as a specific instance of a service occurring at a particular time. The aggregation of all sessions for a service constitutes the service's schedule. |
+| Slot | A time period available for booking by a customer for a service. In appointment services, this encompasses all available booking times. For classes, it refers to all existing sessions open for booking. Refer to appointment schedule for details on appointment services and class schedule for details on classes. |
+| [Service Provider Interface (SPI)](https://dev.wix.com/api/rest/getting-started/service-provider-interface#getting-started_service-provider-interface_introduction) | SPIs are APIs that are defined by Wix, which you can choose to implement. By doing so, you become a service provider. Then, Wix calls your service during a certain flow, waits for your response, and continues the flow with it. For example, as a pricing provider you can offer custom varied pricing options to a Wix site using the [Wix Bookings Pricing Integration REST SPI](https://dev.wix.com/api/rest/wix-bookings/pricing-integration-spi). |
+| Site owner | The owner of the Wix site which has Wix Bookings installed and all [site contributors](https://support.wix.com/en/article/inviting-people-to-contribute-to-your-site). |
```

Comment:
```suggestion
| Site owner | The owner of the Wix site that has Wix Bookings installed and all [site contributors](https://support.wix.com/en/article/inviting-people-to-contribute-to-your-site). |
```
We use "that" without a comma before it.... or ", which" that must have a comma before it. 

https://study.com/learn/lesson/which-vs-that.html#:~:text='That'%20is%20used%20to%20introduce,extra%20information%20about%20a%20noun.